app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,56 @@
+<div class="container">
+  <div class="row">
+    <div class="col-md-3" style="text-align: center; padding: 2rem">
+      <div class="card" style="width: 20rem;">
+        <%= image_tag "recipes.png", class: "card-img-top", style: "width: 100%" %>
+        <div class="card-body">
+          <h4 class="card-title"><%= Recipe.first.name %></h4>
+          <p class="card-text"><%= Recipe.first.instructions %></p>
+          <a href="#" class="btn btn-primary">Check</a>
+        </div>
+      </div>
+    </div>
+    <div class="col-md-3" style="text-align: center; padding: 2rem">
+      <div class="card" style="width: 20rem;">
+        <%= image_tag "recipes.png", class: "card-img-top", style: "width: 100%" %>
+        <div class="card-body">
+          <h4 class="card-title"><%= Recipe.first.name %></h4>
+          <p class="card-text"><%= Recipe.first.instructions %></p>
+          <a href="#" class="btn btn-primary">Check</a>
+        </div>
+      </div>
+    </div>
+    <div class="col-md-3" style="text-align: center; padding: 2rem">
+      <div class="card" style="width: 20rem;">
+        <%= image_tag "recipes.png", class: "card-img-top", style: "width: 100%" %>
+        <div class="card-body">
+          <h4 class="card-title"><%= Recipe.first.name %></h4>
+          <p class="card-text"><%= Recipe.first.instructions %></p>
+          <a href="#" class="btn btn-primary">Check</a>
+        </div>
+      </div>
+    </div>
+    <div class="col-md-3" style="text-align: center; padding: 2rem">
+      <div class="card" style="width: 20rem;">
+        <%= image_tag "recipes.png", class: "card-img-top", style: "width: 100%" %>
+        <div class="card-body">
+          <h4 class="card-title"><%= Recipe.first.name %></h4>
+          <p class="card-text"><%= Recipe.first.instructions %></p>
+          <a href="/recipes/:id" class="btn btn-primary">Check</a>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="row">
```

Comment:
Please no manual splitting. You should have logic responsible for this. What will you do in case when you have 100 recipes? Type them all manually? I don't think so.